src/core/wallet-factory.ts

```diff
@@ -43,3 +45,11 @@ export const getWalletFromRole = async ({ logger, role }) => {
   const user = await User.findOne({ role })
   return WalletFactory({ user, logger })
 }
+
+export const getStringCsvForWallets = async (walletIds: WalletId[]) => {
```

Comment:
Why is new code being added under `core`? This is fragmenting the code base. The code you have here + under the resolver should be unified under an `app/accounts` use case.

Reply:
Done